addons/hr_attendance/static/src/js/kiosk_mode.js

```diff
@@ -55,9 +56,17 @@ var KioskMode = AbstractAction.extend({
 
     destroy: function () {
         core.bus.off('barcode_scanned', this, this._onBarcodeScanned);
+        clearInterval(this._interval);
         clearInterval(this.clock_start);
         this._super.apply(this, arguments);
     },
+
+    call2Db: function () {
```

Comment:
this should be a private function, and with a better name. For example: _callServer, also, it needs documentation, which explains why this is needed